ios-xcframework/fastlane/Fastfile

```diff
@@ -27,20 +27,23 @@ lane :upload_xcframework_to_s3 do |options|
   end
 
   upload_to_a8c_s3(file: xcframework_archive_path, name_on_s3: name)
-  upload_podspec_to_s3(id: last_git_commit[:commit_hash])
+  upload_podspec_to_s3(archive_id: last_git_commit[:commit_hash])
 
   next unless is_ci
 
   tag = ENV.fetch('BUILDKITE_TAG', nil)
 
   next if tag.nil? || tag.empty?
 
-  upload_to_a8c_s3(file: xcframework_archive_path, name_on_s3: "Gutenberg-#{tag}.#{archive_type}")
+  # Tradeoff: We are "wasting" space on S3 by uploading the same archive under different name.
+  # What we get in return is a 1-1 correspondency between archive and podspec, which keeps the setup simpler.
+  upload_to_a8c_s3(file: xcframework_archive_path, name_on_s3: tag)
```

Comment:
Is this change intentional? Does using `tag` as the S3 object name mean the framework URL would be something like `https://cdn.a8c.service/gutenberg-mobile/1.2.3`? I don't think that URL matches the one specified in the podspec file.

Reply:
Good catch @crazytonyli. Funny enough, I noticed that about the same time as you did and was working on a fix: 2e84f32

Reply:
Never mind. I see this part has been updated almost immediately after my comment 😄